Changelog.md

```diff
@@ -86,10 +86,12 @@
     * #350: undefined: build.MultiplePackageError
     * #351: stow away helper files
     * #353: cannot find package "appengine"
+        * Ignore files with the `appengine` build tag
 
 # v33 2015/12/07
 
 * Replace the use of `go list`. This is a large change although all existing tests pass.
+    * Ignore files with the `ignore` build tag. All other build tags are ignored.
```

Comment:
The phrasing is a little confusing. Ignore is used with 2 different meanings.

How about:

> Ignore files with the `ignore` build tag. All other build tags are included and vendored.